engine/src/main/java/org/hibernate/search/bridge/spi/FieldMetadataCreationContext.java

```diff
@@ -33,4 +33,12 @@
 	 * @return this context for fluent API invocations
 	 */
 	FieldMetadataCreationContext sortable(boolean sortable);
+
+	/**
+	 * Give access to properties that are backend specific
```

Comment:
I'll think a differrent word, but we need a term to define these. Should we just refer to index managers?